wayback_discover_diff/web.py

```diff
@@ -81,6 +81,12 @@ def request_url():
             return jsonify({'status': 'error', 'info': 'year param is required.'})
         # validate that year is integer
         int(year)
+        # see if there is an active job for this request
+        pending = APP.celery.control.inspect().active()
+        tasks = list(pending.values())[0]
+        for task in tasks:
+            if task['args'] == "['%s', '%s']" % (url, year):
```

Comment:
Why making an extra string `['%s', '%s']` ? Isn't it simpler to just check:
```
if task['args'][0] == url and task['args'][1] == year:
```

Reply:
Oh, I see that `task['args']` is a string and not a list.

So, its fine. I'm merging this.